packages/atlas/src/providers/nftDialogs.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useContext, useState } from 'react'
+
+import { AcceptBidDialog } from '@/components/_overlays/AcceptBidDialog'
+
+type Dialogs = 'accept-bid' | 'change-price' | 'sign-in'
+
+type ContextValue = {
+  setOpenedDialog: React.Dispatch<React.SetStateAction<Dialogs | undefined>>
+}
+
+export const NftDialogsContext = React.createContext<ContextValue | undefined>(undefined)
```

Comment:
Any reason we need a separate context? Couldn't we use the one we already have for NFT actions?